migrations/versions/0440_new_sms_allowance_n_rate.py

```diff
@@ -0,0 +1,47 @@
+"""
+
+Revision ID: 0440_new_sms_allowance_n_rate
+Revises: 0439_intl_letters_jan_24
+Create Date: 2023-03-06 11:32:20.588364
+
+"""
+
+import uuid
+
+from alembic import op
+
+
+revision = "0440_new_sms_allowance_n_rate"
+down_revision = "0439_intl_letters_jan_24"
+
+new_allowances = {
+    "central": 30000,
+    "nhs_central": 30000,
+    "local": 10000,
+    "nhs_local": 10000,
+    "emergency_service": 10000,
+    "school_or_college": 5000,
+    "other": 5000,
+    "nhs_gp": 0,
+}
+
+new_sms_rate_id = uuid.uuid4()
+
+
+def upgrade():
+    op.execute(
+        "INSERT INTO rates(id, valid_from, rate, notification_type) "
+        f"VALUES('{new_sms_rate_id}', '2024-03-31 23:00:00', 0.0227, 'sms')"
+    )
+
+    for org_type, allowance in new_allowances.items():
+        op.execute(
+            "INSERT INTO default_annual_allowance(id, valid_from_financial_year_start, organisation_type, allowance, notification_type) "
+            f"VALUES('{uuid.uuid4()}', 2024, '{org_type}', {allowance}, 'sms')"
+        )
+
+
+def downgrade():
+    op.execute(f"DELETE FROM rates WHERE id = '{new_sms_rate_id}'")
```

Comment:
This didn't delete the row with a price of 0.0227 for me running flask db downgrade I think it forgets the uuid between different invocations. Perhaps using the valid from to delete the row instead would work.

Reply:
Ooh good catch, you are absolutely right!

Reply:
I fixed it now 😌